TheWarWithin/RogueAssassination.lua

```diff
@@ -256,6 +256,7 @@ local stealth = {
 
     improved_garrote = { "improved_garrote", "sepsis_buff" },
 
+    primal = { "stealth" },
```

Comment:
Since this doesn't actually exist in SimC, I'd rather just use `buff.stealth.up` wherever we're accounting for this bug.  Implementing `stealthed.primal` could cause confusion later.